awl/mslider.cpp

```diff
@@ -234,15 +234,22 @@ void MeterSlider::paintEvent(QPaintEvent* ev)
    	p.setFont(f);
       p.setPen(QPen(Qt::darkGray, 2));
    	QFontMetrics fm(f);
-      int xt = 20 - fm.width("00") - 5;
+      int xt = 20 - fm.width("-00") - 5;
 
       QString s;
-   	for (int i = 10; i < 70; i += 10) {
-      	h  = y1 + lrint(i * mh / range);
-         	s.setNum(i - 10);
-  		p.drawText(xt,  h - 3, s);
-		p.drawLine(15, h, 20, h);
-         	}
+      for (int i = minValue(); i <= maxValue(); i += 10) {
+            h  = y1 + lrint((maxValue() - i) * mh / range);
+            s.setNum(i);
+            if (i > 0)
+                  p.drawText(QRect(xt - 10, h - 4, 20, 8), Qt::AlignRight, QString::fromStdString("+") + s);
+            else if (i == minValue()) {
+                  p.drawText(QRect(xt - 10, h - 4, 20, 8), Qt::AlignRight, QString::fromStdString("-∞"));
```

Comment:
well https://en.wikipedia.org/wiki/C++11#New_string_literals was a bit more informative.  Could write as u"-∞" instead.  (u stores as UTF-16 encoding, while u8 stores as UTF-8 encoding).

https://musescore.org/en/developers-handbook/musescore-coding-rules#File-format says "All code is UTF8, only ASCII to be used in C++ code for max. portability." so to me that means that it is ok to have unicode inside of a string literal, but that unicode shouldn't be used for things like function and variable names.